oauthlib/oauth1/rfc5849/endpoints/__init__.py

```diff
@@ -1,7 +1,8 @@
+from .access_token import AccessTokenEndpoint
+from .authorization import AuthorizationEndpoint
 from .base import BaseEndpoint
 from .request_token import RequestTokenEndpoint
-from .authorization import AuthorizationEndpoint
-from .access_token import AccessTokenEndpoint
 from .resource import ResourceEndpoint
 from .signature_only import SignatureOnlyEndpoint
-from .pre_configured import WebApplicationServer
+
+from .pre_configured import WebApplicationServer  # isort:skip
```

Comment:
circular import if `import .pre_configured` is before `import .request_token` line